src/nvim/eval.c

```diff
@@ -15239,9 +15242,20 @@ static void set_qf_ll_list(win_T *wp, typval_T *list_arg, typval_T *action_arg,
         action = *act;
     }
 
-    if (l != NULL && set_errorlist(wp, l, action,
-            (char_u *)(wp == NULL ? "setqflist()" : "setloclist()")) == OK)
+    if (title_arg->v_type == VAR_STRING) {
```

Comment:
I think this should be `!= VAR_UNKNOWN` and not `== VAR_STRING`. Function should error out if title argument is not a string, not silently ignore the argument. Do not forget a test for this behaviour.